docs/standard/microservices-architecture/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -61,15 +63,17 @@ services.AddHttpClient<IBasketService, BasketService>();
 services.AddHttpClient<IOrderingService, OrderingService>();
 ```
 
-Just by adding your typed client classes with AddHttpClient(), whenever you use the `HttpClient` object that will be injected to your class through its constructor, that `HttpClient` object will be using all the configuration and policies provided. In the next sections, you will see those policies like Polly’s retries or circuit-breakers.
+Registering the client services as shown in the code above, makes the `DefaultClientFactory` create an `HttpClient` configured specifically for each service, as we'll see in just a moment.
+
+Just by registering your client classes with AddHttpClient(), whenever you use the `HttpClient` object injected to your class through its constructor, that `HttpClient` object will be using all the configuration and policies provided upon registration. In the next sections, you will see those policies like Polly’s retries or circuit-breakers.
```

Comment:
backtick around AddHttpClient too

this first sentence is a bit hard to parse

you will -> you'll see

Reply:
Simplified it a bit.